src/drivers/distance_sensor/rds02uf/module.yaml

```diff
@@ -0,0 +1,46 @@
+module_name: Benewake RDS02UF Rangefinder
+serial_config:
+    - command: rds02uf start -d ${SERIAL_DEV} -R SENS_RDS02UF_ROT
+      port_config_param:
+        name: SENS_RDS02_CFG
+        group: Sensors
+
+parameters:
+    - group: Sensors
+      definitions:
+        SENS_RDS02_ROT:
+            description:
+                short: Distance Sensor Rotation
+                long: |
+                    Distance Sensor Rotation as MAV_SENSOR_ORIENTATION enum
+
+            type: enum
+            values:
+                25: ROTATION_DOWNWARD_FACING
+                24: ROTATION_UPWARD_FACING
+                4: ROTATION_BACKWARD_FACING
+                0: ROTATION_FORWARD_FACING
+                6: ROTATION_LEFT_FACING
+                2: ROTATION_RIGHT_FACING
+            reboot_required: true
+            default: 25
+
+        SENS_RDS02_MAXD:
```

Comment:
Why should this be user configurable? Isn't it a property of the sensor?

Reply:
Similar to TFMINI products, they have different series of products that can use the same driver code, the main difference is the ranging range

Reply:
Can it be detected?

Reply:
If at all possible I think we should be setting the sensor properties automatically based on the actual model rather than leaving it open for possible user misconfiguration.

Reply:
Thank you for your suggestion. I have linked the range parameters of the sensor to the sensor type. The code changes have been resubmitted. The pull request is here: #21379